src/feed/impl/TransactionFeed.hpp

```diff
@@ -182,12 +182,14 @@ class TransactionFeed {
      * @param txMeta The transaction and metadata.
      * @param lgrInfo The ledger header.
      * @param backend The backend.
+     * @param networkID The network ID.
      */
     void
     pub(data::TransactionAndMetadata const& txMeta,
         ripple::LedgerHeader const& lgrInfo,
         std::shared_ptr<data::BackendInterface const> const& backend,
-        std::shared_ptr<data::AmendmentCenterInterface const> const& amendmentCenter);
+        std::shared_ptr<data::AmendmentCenterInterface const> const& amendmentCenter,
+        uint32_t const networkID = 0);
```

Comment:
Probably better to not put a default value here, because having default value we could miss a place where `networkID` should be passed and it will result in sending incorrect data.